spdx/v2/common/external.go

```diff
@@ -4,6 +4,10 @@ package common
 
 // Constants for various string types
 const (
+	CategorySecurity       = "SECURITY"
+	CategoryPackageManager = "PACKAGE-MANAGER"
+	TypePersistentIdSwh    = "swh"
```

Comment:
looks good - can we create a comment to show which section of the spec they come from? (like below). After this PR would you be able to help with a follow-up PR to change all uses of the categories to use the const as well? :D

Reply:
What section do you think `TypePersistentIdSwh` should go into

Reply:
I think the Categories are part of [external references](https://spdx.github.io/spdx-spec/v2.3/package-information/#721-external-reference-field). `swh` is from [Annex F](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f4-persistent-id).

Reply:
Thanks, I have fixed this